src/app/reporting/reporting.c

```diff
@@ -138,8 +157,11 @@ void emberAfPluginReportingInitCallback(void)
     {
         EmberAfPluginReportingEntry entry;
         emAfPluginReportingGetEntry(i, &entry);
+#pragma GCC diagnostic push
```

Comment:
Need to add these to avoid uninitialized and maybe-initialized warnings being treated as errors.

Reply:
That's annoying.  I wish compilers understood the "out param here!" pattern this is using...

I guess we could initialize the struct with `{}` if we really care; might be less noisy than having to do this all over the place  Followup for that, maybe?

Reply:
Sure will do a follow up